templates.json

```diff
@@ -702,9 +690,30 @@
         "path": "editor-extension-collection"
       }
     ],
-    "organizationBetaFlags": [
-      "editor_extension_collection"
-    ]
+    "organizationBetaFlags": ["editor_extension_collection"]
+  },
+  {
+    "identifier": "discount_details_ui",
```

Comment:
Assuming that this is the --template flag, can we name it: `discount_function_metafields`